json_serializable/test/integration/json_test_common.dart

```diff
@@ -23,8 +23,13 @@ enum StatusCode {
   success,
   @JsonValue(404)
   notFound,
+
+  /// Intentionally using a non-int value to validate heterogeneous
```

Comment:
[nit] This is an implementation comment, not a doc comment. Use 2 slashes.

Reply:
ack